serpy/fields.py

```diff
@@ -110,20 +183,28 @@ def do_minus(self, foo_obj):
                 return foo_obj.bar - foo_obj.baz
 
         foo = Foo(bar=5, baz=10)
-        FooSerializer(foo).data
+        FooSerializer(foo).representation
         # {'plus': 15, 'minus': -5}
 
     :param str method: The method on the serializer to call. Defaults to
         ``'get_<field name>'``.
     """
     getter_takes_serializer = True
+    setter_takes_serializer = True
 
-    def __init__(self, method=None, **kwargs):
+    def __init__(self, getter=None, setter=None, **kwargs):
         super(MethodField, self).__init__(**kwargs)
-        self.method = method
+        self.getter_method = getter
+        self.setter_method = setter
 
     def as_getter(self, serializer_field_name, serializer_cls):
-        method_name = self.method
+        method_name = self.getter_method
         if method_name is None:
             method_name = 'get_{0}'.format(serializer_field_name)
-        return getattr(serializer_cls, method_name)
+        return getattr(serializer_cls, method_name, None)
```

Comment:
i would rather not specify the default to getattr and have these methods fail if method_name doesn't exist